tools/SaveModules.ps1

```diff
@@ -0,0 +1,106 @@
+$AzureMajorVersion = "0"
+
+$AzureRMModules = @(
+  "AzureRM.ApiManagement",
+  "AzureRM.Automation",
+  "AzureRM.Backup",
+  "AzureRM.Batch",
+  "AzureRM.Compute",
+  "AzureRM.DataFactories",
+  "AzureRM.Dns",
+  "AzureRM.HDInsight",
+  "AzureRM.Insights",
+  "AzureRM.KeyVault",
+  "AzureRM.Network",
+  "AzureRM.OperationalInsights",
+  "AzureRM.RedisCache",
+  "AzureRM.Resources",
+  "AzureRM.SiteRecovery",
+  "AzureRM.Sql",
+  "AzureRM.Storage",
+  "AzureRM.StreamAnalytics",
+  "AzureRM.Tags",
+  "AzureRM.TrafficManager",
+  "AzureRM.UsageAggregates",
+  "AzureRM.Websites"
+)
+
+$AzureRMPath = "$PSScriptRoot\..\src\Package\Debug\ResourceManager\AzureResourceManager"
+
+$AzureSMPath = "$PSScriptRoot\..\src\Package\Debug\ServiceManagement"
+
+function Save-ModuleWithVersionCheck([string]$Name,[string]$MajorVersion,[string]$Repository,[string]$Path)
+{
+  $_MinVer = "$MajorVersion.0.0.0"
+  $_MaxVer = "$MajorVersion.9999.9999.9999"
+  $script:InstallCounter ++
+  try {
+    Save-Module -Name $Name -Repository $Repository -MinimumVersion $_MinVer -MaximumVersion $_MaxVer -ErrorAction Stop -Path $Path
+    $versionFolder = Get-ChildItem "$Path\$Name\" | Select-Object -First 1
+    Get-ChildItem $versionFolder.FullName | ForEach { Move-Item -Path $_.FullName -Destination "$Path\$Name\" }
```

Comment:
Because the files are saved under subfolder that has the version and is needed to eliminate that extra layer